docs/nodes/hugging_face/hugging_face_install.md

```diff
@@ -0,0 +1,143 @@
+
+### 1. Create a new account on Hugging Face
+
+1. Go to [https://huggingface.co/](https://huggingface.co/)
+1. Click **Sign Up** in the top-right corner
+1. Complete the verification step to prove you're not a robot
+1. Create and record your credentials
+1. Complete the email verification process
+
+<p align="center">
+  <img src="/assets/img/hugging_face/00_HF_MainPage.png" alt="HF Site" width="500"/>
+</p>
+
+<p align="center">
+  <img src="/assets/img/hugging_face/01_HF_signup.png" alt="Signup" width="300"/>
+</p>
+
+### 2. Access Your Account Settings
+
+1. Log in to your Hugging Face account
+1. Click on your profile icon in the top right corner
+1. Select **Settings** from the dropdown menu (or go directly to [Settings](https://huggingface.co/settings/profile/))
+
+<p align="center">
+  <img src="/assets/img/hugging_face/02_HF_Settings.png" alt="Settings" width="500"/>
+</p>
+
+### 3. Create an Access Token
+
+!!! warning "Email Verification Required"
+
+    If you encounter issues during token creation, ensure you've verified your email address. Complete the verification process before continuing.
+
+1. Navigate to **Access Tokens** in the settings menu
+1. Click **Create new token** in the top right area
+1. Select **Read** as the token type for basic access
+1. Give your token a descriptive name
+1. Copy and securely store your token
+
+<p align="center">
+  <img src="/assets/img/hugging_face/03_HF_AccessTokens.png" alt="Access Tokens" width="500"/>
+</p>
+
+<p align="center">
+  <img src="/assets/img/hugging_face/04_HF_TokenRead.png" alt="Token Read" width="500"/>
+</p>
+
+<p align="center">
+  <img src="/assets/img/hugging_face/05_HF_SaveToken.png" alt="Save Token" width="500"/>
+</p>
+
+!!! danger "Security Notice"
+
+    Your access token is a personal identifier for requests made to Hugging Face services. Never share it with anyone, and take precautions to avoid displaying it in screenshots, videos, or during screen-sharing sessions.
+
+## Install Required Files
+
+Now that you have a token associated with your account, you can install the Hugging Face CLI (Command Line Interface) to interact with Hugging Face from the command line.
+
+### 1. Install the Hugging Face CLI
+
+Open a terminal and run:
+
+```bash
+pip install -U "huggingface_hub[cli]"
+```
+
+For more information, visit the [official CLI documentation](https://huggingface.co/docs/huggingface_hub/main/en/guides/cli).
+
+### 2. Login with Your Token
+
+In your terminal, authenticate with your access token:
+
+```bash
+huggingface-cli login
+```
+
+You'll be prompted to enter your token.
+
+### 3. Install Required Models
+
+!!! note "Download Time"
+
+    These model downloads may collectively take 30-50 minutes to complete, depending on your internet connection speed.
+
+Install the following models for use with different Griptape Nodes:
```

Comment:
Where should we recommend they download these?

I just ran all the install commands in my home directory - should we have them put these in `~/GriptapeNodes`? Where do they look by default?

Reply:
Ah, it looks like they get installed into your local hugging face cache dir.

We should add a note:

Models will be downloaded into your Hugging Face Hub Cache Directory. To see where this is, you can use the `huggingface-cli`, and look for `HF_HUB_CACHE`.

Example:

```bash
huggingface-cli env
```

```bash
- HF_HUB_CACHE: /Users/jason/.cache/huggingface/hub
```

To learn more about how files are cached, please refer to the Hugging Face [caching guide]([caching guide](https://huggingface.co/docs/huggingface_hub/main/en/guides/manage-cache)).